src/cadence.py

```diff
@@ -1330,7 +1330,7 @@ def jackStarted(self):
             self.label_jack_realtime_ico.setPixmap(self.pix_cancel)
 
         self.label_jack_dsp.setText("%.2f%%" % self.m_last_dsp_load)
-        self.label_jack_xruns.setText(str(self.m_last_xruns))
+        self.label_jack_xruns.setText(str(int(self.m_last_xruns)))
```

Comment:
There is also this line https://github.com/falkTX/Cadence/blob/6e8cd944812c247c4b52a2ba15f0db93bc1d42eb/src/cadence.py#L2337 that needs to change.

Reply:
Fixed, thanks.